tests/index.html

```diff
@@ -134,7 +126,7 @@
           testsTotal = testsPassed = testsFailed = 0;
 
           if (QUnit.urlParams.hideskipped) {
-            $('#qunit-tests').addClass('hideskipped');
+            document.getElementById('qunit-tests').classList.add('hideskipped');
```

Comment:
This was the reason for failing FF and Browserstack tests! They used `hideskipped` in `testem.ci-browsers.js`, which was still jQuery based here. Still don't understand why this gave us these obscure error messages...

Reply:
Good catch!!!